mkdocs.yml

```diff
@@ -165,5 +165,6 @@ nav:
       - developers/github-selfhosted-setup.md
       - developers/project-types.md
       - developers/release-management.md
-      - developers/styleguide.md
```

Comment:
We'll want to add a redirect for this after it goes in.

Reply:
Is that something I can do here?

Reply:
Not AFAIK. I've been doing redirects in readthedocs. And of course this might break stable while latest is out there. I don't think it's all that important, but let's remember we have a task after it's pulled.